src/workspace.coffee

```diff
@@ -506,6 +506,12 @@ class Workspace extends Model
   #
   # Returns a {Disposable} on which `.dispose()` can be called to remove the
   # opener.
+  #
+  # Note that the opener will be called if and only if the URI is not already open
+  # in the current pane. The searchAllPanes flag expands the search from the
+  # current pane to all panes. If the file path is converted from a standard path, 
```

Comment:
Starting with `If the file path`:

I think this might be confusing for someone coming to this documentation without being in your specific situation. How about replacing it with something like:

> If you wish to open a view of a different type for a file that is already open, consider changing the protocol of the URI. For example, perhaps you wish to preview a rendered version of the file `/foo/bar/baz.quux` that is already open in a text editor view. You could signal this by calling {Workspace::open} on the URI `quux-preview://foo/bar/baz.quux`. Then your opener can check the protocol for `quux-preview` and only handle those URIs that match.

What do you think?